doc/source/reference/arrays.classes.rst

```diff
@@ -173,7 +173,7 @@ NumPy provides several hooks that classes can customize:
 
    -  ``func`` is an arbitrary callable exposed by NumPy's public API,
       which was called in the form ``func(*args, **kwargs)``.
-   -  ``types`` is a `collection <collections.abc.Collection>`_
```

Comment:
This ought to be handled by intersphinx.

Reply:
Perhaps `python:collections.abc.Collection`, or `` :class:`collection <collections.abc.Collection>` `` is needed here.

Reply:
```
-  ``types`` is a collection :py:class:`collections.abc.Collection`
```
Thanks for the suggestion, putting it in.